### PR TITLE
Docs:Add an FAQ item about how to configure Skywalking Tracing.

### PR DESCRIPTION
**Pull request by Frapschen**
Commit Message:
Add an FAQ item about how to configure Skywalking Tracing.
Docs Changes:

- add an item in _docs/root/faq/overview.rst_

- add a file _docs/root/faq/configuration/skywalking_tracing.rst_

### CODE REVIEW

**Review by jmarantz — DISMISSED** [review]
of course the github preview doesn't work for these rst files, but otherwise I think this is fine.

**Comment by jmarantz** [comment]
oh it looks like this failed CI for docs generation; can you take a deeper look and ping back?

/wait

**Comment by Frapschen** [comment]
I looked at the Azure PipIines and only finded an error code without any other userful message.

https://dev.azure.com/cncf/envoy/_build/results?buildId=63564&view=logs&j=91e633a5-4907-5da1-5862-92a79fe2387a&t=f29e24e7-a093-5724-80bd-d79975d268cb&l=4381

**Comment by Frapschen** [comment]
I still haven't found anything to solve the popline problem

**Comment by jmarantz** [comment]
This is what I found scanning through the CI logs:
```
2021-01-13T10:34:15.4488273Z reading sources... [100%] version_history/version_history
2021-01-13T10:34:15.4593108Z 
2021-01-13T10:34:15.4594181Z /source/generated/rst/faq/configuration/skywalkin_tracing.rst:4: WARNING: Title underline too short.
2021-01-13T10:34:15.4594944Z 
2021-01-13T10:34:15.4595686Z How do I configure skywalking tracing?
2021-01-13T10:34:15.4596502Z ==================================
2021-01-13T10:34:15.4598196Z /source/generated/rst/faq/overview.rst:56: WARNING: toctree contains reference to nonexisting document 'faq/configuration/skywalking_tracing'
2021-01-13T10:34:15.4599585Z looking for now-outdated files... none found
2021-01-13T10:34:15.5204631Z pickling environment... done
2021-01-13T10:34:15.5206746Z checking consistency... done
2021-01-13T10:34:15.5208063Z /source/generated/rst/faq/configuration/skywalkin_tracing.rst: WARNING: document isn't included in any toctree
2021-01-13T10:34:15.5261080Z preparing documents... done
```

**Comment by jmarantz** [comment]
Please do not force-push with Envoy and github. It causes issues with the review comments. Just 'merge main'.

**Comment by jmarantz** [comment]
In any case it looks like the latest update fixed the 'docs' precheck. I still am not sure though if there's any substantive documentation this is adding.

Is that already in the tree?

/wait

**Comment by Frapschen** [comment]
This add PR just adds a link to a sandbox of skywlking tracing, like this file $configuration/zipkin_tracing. rts$

**Comment by mattklein123** [comment]
+1 that I don't think this adds enough value to be an independent FAQ. How about this:

Make the Zipkin FAQ entry a generic entry on "how to configure tracing" and then link to the tracing docs?

**Comment by Frapschen** [comment]
sounds good

File: docs/root/faq/configuration/tracing.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. _common_configuration_tracing:
+
+How do I configure tracing?
```
Comment (mattklein123) [inline docs/root/faq/configuration/tracing.rst:3]: Can you also link to the generic tracing documentation? https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing

Thank you

/wait

Comment (Frapschen) [reply docs/root/faq/configuration/tracing.rst:3]: ok， I will try

**Pull request closed without merging**